specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -154,6 +163,33 @@ export function TreeRow({
           const stats =
             typeof nodeStats === 'object' &&
             formatTreeStats(nodeStats, row.children === 0);
+          const table = genericTables[treeName] as SpecifyTable<AnyTree>;
+          const [resource] = useLiveState<
+            SpecifyResource<AnySchema> | undefined
+          >(
+            React.useCallback(() => {
```

Comment:
sorry, but you keep making the same mistake ---- you can't call react hooks anywhere but on the top level of a component
you can't call it inside of callbacks, inside of loops, inside of conditions!

please read this article carefully - https://legacy.reactjs.org/docs/hooks-rules.html#:~:text=Only%20Call%20Hooks%20at%20the%20Top%20Level
feel free to also research online how react hooks work under the hood so that you can understand why such restriction exists

Reply:
instead, if you need some hook result to be used inside your loop, then either move the hook out of the loop, OR, if your hook relies on some data specific to that loop iteration, then move the loop body into it's own component and then inside of that component you can call your hooks at the top level